ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -632,6 +653,41 @@ objects:
         TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create cluster 2>&1 | grep --line-buffered -v password &
         wait "$!"
 
+        if [[ "${CLUSTER_TYPE}" == "gcp" ]] && [[ "${CLUSTER_ENABLE_NESTED_VIRT-false}" == "true" ]]; then
```

Comment:
I've always seen the `-` spelled `:-` but I guess this works?

Reply:
Bash manual says "... When not performing substring expansion, using the forms documented below (e.g., :-), bash tests for a parameter that is unset or null.  Omitting the colon results in a test only for a parameter that is unset. ..." so I think it does not make a difference whether we use the colon here.

Reply:
Sigh. Bash is a mystery. I heared that the implementation is .. gorgeous ..